build/opnsense.json

```diff
@@ -58,19 +53,17 @@
         [ "modifyvm", "{{.Name}}", "--memory", "1024" ],
         [ "modifyvm", "{{.Name}}", "--cpus", "1" ],
         [ "modifyvm", "{{.Name}}", "--boot1", "disk" ],
-        [ "modifyvm", "{{.Name}}", "--boot2", "dvd" ],
         [ "modifyvm", "{{.Name}}", "--audio", "none" ],
```

Comment:
dont we need this for an iso based installation? Does it help removing it (for what?)